tests/test_table.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+from roulette import Table, Bet, Outcome, InvalidBet
+
+
+class TestTable(TestCase):
+    def setUp(self):
+        self.oc1 = Outcome("Black", 1)
+        self.oc2 = Outcome("1", 35)
+        self.bet1 = Bet(5, self.oc1)
+        self.bet2 = Bet(300, self.oc2)
+        self.table = Table()
+
+    def test_placeBet_adds_bets_to_bets_list(self):
+        bet3 = Bet(30, self.oc1)
+        self.table.placeBet(bet3)
+        self.assertIn(bet3, self.table.bets)
+
+    def test_isValid_raises_exception_for_bets_lower_than_minimum_bet_amount(self):
+        self.table.placeBet(self.bet1)
+        with self.assertRaises(InvalidBet):
+            self.table.isValid()
+
+    def test_isValid_raises_exception_when_bet_amounts_cross_table_limit(self):
```

Comment:
The exception raised from this test is due to min amount violation, not due to sum of bet crosses table limit.

This was realized after looking at the coverage test report.

<img width="1426" alt="Screenshot 2024-01-22 at 11 47 54 AM" src="https://github.com/FarzanRashid/Object-Oriented-Design/assets/8214491/b7fd7758-b1e9-4e86-a31e-4c78638c30e2">

The lines [595-597](https://github.com/FarzanRashid/Object-Oriented-Design/pull/6/files#diff-79f77304c81d96f0c53bee4669638da2afcecac1255169a758155a78d02ad064R595-R597) is supposed to be covered by test

Reply:
Fixed. Thanks.